opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpan.java

```diff
@@ -127,6 +125,18 @@ public String getBaggageItem(String key) {
     }
 
     @Override
+    public final Span log(Field... fields) {
+        return log(System.nanoTime() / 1000, fields);
```

Comment:
probably setting a bad example with this, from https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#nanoTime():

> This method can only be used to measure elapsed time and is not related to any other notion of system or wall-clock time. The value returned represents nanoseconds since some fixed but arbitrary origin time (perhaps in the future, so values may be negative). The same origin is used by all invocations of this method in an instance of a Java virtual machine; other virtual machine instances are likely to use a different origin.